assests/css/index.css

```diff
@@ -1,2 +1,103 @@
 @import "./reset.css";
-@import "./variables.css";
\ No newline at end of file
+@import "./variables.css";
+
+body {
+  background-image: linear-gradient(
+      var(--darkColorHover),
+      var(--darkColorHover)
+    ),
+    url(../images/background.jpg);
+  background-size: cover;
+  min-height: 100vh;
+  color: var(--lightColor);
+  align-items: center;
+  font-family: var(--primaryFont);
+}
+
+.header {
+  width: 100%;
```

Comment:
why do we need this?

Reply:
WOW I deleted and it worrrrrkeeeeeedddddd
Thanks